addons/website_crm_partner_assign/static/src/interactions/crm_partner_assign.js

```diff
@@ -0,0 +1,158 @@
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+
+import { _t } from "@web/core/l10n/translation";
+import { parseDate, formatDate, serializeDate } from "@web/core/l10n/dates";
+
+const { DateTime } = luxon;
+
+const parse_date = function (value) {
+    var date = parseDate(value);
+    if (!date.isValid || date.year < 1900) {
+        return false;
+    }
+    return serializeDate(date);
+}
```

Comment:
What about defining it inside the method where it is used ? And maybe find a name without `_`.